src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/CalculationStatus.kt

```diff
@@ -0,0 +1,5 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model
```

Comment:
instead of placing this in the models package, consider a new package called `enumerations`

Reply:
agreed thanks - changes done